### DISCUSSION TITLE
be经常自己挂掉

### DISCUSSION THREAD

**Discussion by JavaDevelopmenter** [apache/doris#7456] [A - General / Q&A]
*** Aborted at 1640074624 (unix time) try "date -d @1640074624" if you are using GNU date ***
PC: @          0x30dadd4 std::thread::join()
*** SIGSEGV (@0x0) received by PID 13930 (TID 0x7fae1d5a1700) from PID 0; stack trace: ***
    @          0x201ac02 google::(anonymous namespace)::FailureSignalHandler()
    @     0x7fae4defb5f0 (unknown)
    @          0x30dadd4 std::thread::join()
    @          0x19417f9 doris::OlapScanNode::close()
    @          0x193ca6f doris::TopNNode::open()
    @          0x14a4ea4 doris::PlanFragmentExecutor::open_internal()
    @          0x14a998f doris::PlanFragmentExecutor::open()
    @          0x142045e doris::FragmentExecState::execute()
    @          0x14239e6 doris::FragmentMgr::_exec_actual()
    @          0x142b73b std::_Function_handler<>::_M_invoke()
    @          0x1592db3 doris::ThreadPool::dispatch_thread()
    @          0x158cefc doris::Thread::supervise_thread()
    @     0x7fae4def3e65 start_thread
    @     0x7fae4d91a88d __clone
start time: Tue Dec 21 16:22:26 CST 2021
*** Aborted at 1640082364 (unix time) try "date -d @1640082364" if you are using GNU date ***
PC: @          0x30dadd4 std::thread::join()
*** SIGSEGV (@0x0) received by PID 1793 (TID 0x7fca5656e700) from PID 0; stack trace: ***
    @          0x201ac02 google::(anonymous namespace)::FailureSignalHandler()
    @     0x7fcaa07e35f0 (unknown)
    @          0x30dadd4 std::thread::join()
    @          0x19417f9 doris::OlapScanNode::close()
    @          0x14a3c1c doris::PlanFragmentExecutor::close()
    @          0x1420474 doris::FragmentExecState::execute()
    @          0x14239e6 doris::FragmentMgr::_exec_actual()
    @          0x142b73b std::_Function_handler<>::_M_invoke()
    @          0x1592db3 doris::ThreadPool::dispatch_thread()
    @          0x158cefc doris::Thread::supervise_thread()
    @     0x7fcaa07dbe65 start_thread
    @     0x7fcaa020288d __clone
start time: Tue Dec 21 18:39:38 CST 2021
*** Aborted at 1640084545 (unix time) try "date -d @1640084545" if you are using GNU date ***
PC: @          0x30dadd4 std::thread::join()
*** SIGSEGV (@0x0) received by PID 24722 (TID 0x7fde5e36e700) from PID 0; stack trace: ***
    @          0x201ac02 google::(anonymous namespace)::FailureSignalHandler()
    @     0x7fdea86cd5f0 (unknown)
    @          0x30dadd4 std::thread::join()
    @          0x19417f9 doris::OlapScanNode::close()
    @          0x1a056c8 doris::PartitionedAggregationNode::open()
    @          0x14a4ea4 doris::PlanFragmentExecutor::open_internal()
    @          0x14a998f doris::PlanFragmentExecutor::open()
    @          0x142045e doris::FragmentExecState::execute()
    @          0x14239e6 doris::FragmentMgr::_exec_actual()
    @          0x142b73b std::_Function_handler<>::_M_invoke()
    @          0x1592db3 doris::ThreadPool::dispatch_thread()
    @          0x158cefc doris::Thread::supervise_thread()
    @     0x7fdea86c5e65 start_thread
    @     0x7fdea80ec88d __clone
start time: Tue Dec 21 19:03:30 CST 2021
*** Aborted at 1640084717 (unix time) try "date -d @1640084717" if you are using GNU date ***
PC: @          0x30dadd4 std::thread::join()
*** SIGSEGV (@0x0) received by PID 7524 (TID 0x7f9eb637f700) from PID 0; stack trace: ***
    @          0x201ac02 google::(anonymous namespace)::FailureSignalHandler()
    @     0x7f9ef7cf75f0 (unknown)
    @          0x30dadd4 std::thread::join()
    @          0x19417f9 doris::OlapScanNode::close()
    @          0x1a056c8 doris::PartitionedAggregationNode::open()
    @          0x14a4ea4 doris::PlanFragmentExecutor::open_internal()
    @          0x14a998f doris::PlanFragmentExecutor::open()
    @          0x142045e doris::FragmentExecState::execute()
    @          0x14239e6 doris::FragmentMgr::_exec_actual()
    @          0x142b73b std::_Function_handler<>::_M_invoke()
    @          0x1592db3 doris::ThreadPool::dispatch_thread()
    @          0x158cefc doris::Thread::supervise_thread()
    @     0x7f9ef7cefe65 start_thread
    @     0x7f9ef771688d __clone

**Comment by ChPi** [comment]
看着是join查询的挂掉
看看系统日志，是不是oom，被系统kill了

**Reply by JavaDevelopmenter** [reply]
查询空的单表的时候也会挂掉

**Reply by JavaDevelopmenter** [reply]
查了下系统日志, 不是系统kill的

**Comment by cooldwind** [comment]
dmesg -T ，看下

**Comment by dataalive** [comment]
可以尝试下最新的1.0.0-rc02，类似异常栈应该修复了。
